.github/workflows/ci.yaml

```diff
@@ -144,7 +144,7 @@ jobs:
         run: mix compile --warnings-as-errors
 
       - name: Test
-        run: mix coveralls.github
+        run: mix coveralls.github --parallel
```

Comment:
Had to add this due to related error in [previous](https://github.com/stordco/buffy/actions/runs/7200078870/job/19616292552) runs.